jsonpointer.js

```diff
@@ -77,13 +77,17 @@ function set (obj, pointer, value) {
 }
 
 function compile (pointer) {
+  var source = pointer.toString()
```

Comment:
I don't think that this line is needed.

Reply:
No worries @marcbachmann 

I'll update the PR accordingly and add a test.

Cheers,
Damo.